### PR TITLE
wip added lit review but can't figure out rst errors

### PR DESCRIPTION
**Pull request by jzf2101**
I'm stumped on how to fix these indent errors in the pdf

### CODE REVIEW

**Comment by jzf2101** [comment]
Also w.r.t.  to calling ourselves 

> the first free, open-source, and massively publicly-available tool for
> easily creating sharable, interactive, reproducible environments in the cloud.

CodaLab https://github.com/codalab is also free, open-source and massively publicly available.  The big distinction, afaik, is that they want you use their types of formats.  Maybe that makes it less easy but I think we may want to modify language.

**Comment by mpacer** [comment]
I would greatly prefer if our references were using a bib file.

**Comment by jzf2101** [comment]
Moving as appropriate and migrating links to bib too.

File: papers/project_jupyter/paper.rst [hunk 1]
```diff
@@ -138,13 +141,26 @@ itself fully open-source for others to use.
    the user to install dependencies.
 
 
-The first iteration of Binder was released in 2016, and provided a prototype
+Binder continues in the tradition of promoting "the complete software
+development environment and the complete set of instructions which generated
+the figures" [Buckheit95]_ by effortlessly providingthese tools to the general
```
Comment (choldgraf) [inline papers/project_jupyter/paper.rst]: providing these (add space)

**Comment by choldgraf** [comment]
this looks really nice, thanks for the refs @jzf2101 ! Ping when you've got an updated version of the PR up and we can merge!

**Comment by choldgraf** [comment]
(don't worry too much about formatting, those issues can be touched up in subsequent iterations)

**Comment by jzf2101** [comment]
I can't build now because I don't have the `.cls` for some reason

**Comment by choldgraf** [comment]
I'm gonna merge because this look great - thanks a bunch of the refs and edits 💯

**Pull request merged**